invenio_rdm_records/records/api.py

```diff
@@ -99,6 +99,7 @@ class CommonFieldsMixin:
             attrs=['id', 'title', 'props.type', 'props.subtype'],
             pid_field=Vocabulary.pid.with_type_ctx('resourcetypes'),
             cache_key='resource_type',
+            value_check=dict(props=dict(relation_type='depositable')),
```

Comment:
So shouldn't the resource_types fixtures (https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml) have these changes?

Reply:
In principle all of the `resource_types` we have there should be linkable AND depositable so we don't really need to update it, we should update it if we want one of them to be only linkable OR depositable .

Reply:
And we will document that somewhere.